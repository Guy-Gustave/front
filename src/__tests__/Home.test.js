import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter, Link } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <div className="App">
        <BrowserRouter>
        <Link to="/" role="button">Home</Link>
        </BrowserRouter>
      </div>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
