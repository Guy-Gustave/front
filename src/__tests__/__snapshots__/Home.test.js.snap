// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <a
    href="/"
    onClick={[Function]}
    role="button"
  >
    Home
  </a>
</div>
`;
